mac/RT1.MAC

```diff
@@ -41,15 +43,18 @@
 .* 9.  RT1 OK    - INC TEST COUNT BY 1                                    
 .* 9.  RT1 ABORT - ABORT WITH TOTAL TESTS TO FAILURE AND USER 101 DUMP
 .*10.  RT1 END   - DISPLAY TOTAL TESTS AND ENDED OK WTO
+.*11   RT1 ABEND - Abend with specified abend code                       #540
```

Comment:
A few things.
1. There are two item 9s. Renumber last three items 10, 11 and 12. (Has nothing to do with #540.)
2. For the second item 9 (ABORT): change "USER 101 DUMP" to "USER ABEND". DUMP not specified on the ABEND invocation.
3. For the ABEND line, make it all caps.

Reply:
1. done.
2. leaving mixed case as is.